azure-pipeline.yaml

```diff
@@ -67,23 +72,44 @@ stages:
   # Only allow one instance of this stage to run at a time to avoid resource contention issues with the targets used in testing.
   lockBehavior: sequential
   dependsOn:
-  - PrepareBuild
+  - BuildInformation
   jobs:
-  - job: UnitTestforPython
+  - job: SanityTests
     continueOnError: false
     steps:
     - template: /eng/pipeline/python/templates/setup-venv.yml@DevCentral/ni-central
       parameters:
         venvName: test_venv
         ${{ if ne(parameters.mobilize_ats_pr_build, ' ') }}:
           pipPkgs:
-          - rtos-oetest-locked==$(mobilize_ats_pr_build)
+          - rtos-oetest-locked==${{ parameters.mobilize_ats_pr_build }}
           usePRServer: true
         ${{ else }}:
           pipPkgs:
           - rtos-oetest-locked
 
-    - script: |
+    - powershell: |
+        $(test_venv.bin_dir)/mako-render $(test_venv.bin_dir)\..\Lib\site-packages\rtos_oetest\mobilize_plans\rtos_oetest\snac\pr\snac_pr_params.yaml.makoo --var majmin_ver="$(Version.Distribution)" --var REPO_VERSION="$(REPO_EXTERNAL_VERSION)" --var REPO_ROOT="$(Build.SourcesDirectory)" --var REPO_EXTERNAL_VERSION_NOSPACE="$(REPO_EXTERNAL_VERSION_NOSPACE)" --output-file $(ParamsOutputFile)
+      displayName: Mako Render Mobilize Params File
+
+    - powershell: |
+        cat $(ParamsOutputFile)
+      displayName: Display Mobilize Params File
+
+    - powershell: |
+        cat $(test_venv.bin_dir)\..\Lib\site-packages\rtos_oetest\mobilize_plans\rtos_oetest\snac\pr\snac_pr_provision_and_test.yaml.mako
+      displayName: Display Pre-Compiled Mobilize File
+
+    - powershell: |
+        $(test_venv.bin_dir)/mobilize -vv execute -p $(test_venv.bin_dir)\..\Lib\site-packages\rtos_oetest\mobilize_plans\rtos_oetest\snac\pr\snac_pr_provision_and_test.yaml.mako -a $(ParamsOutputFile) -o $(CompiledMobilizePlan)
+      displayName: Execute Mobilize Plan
+
+    - powershell: |
+        cat $(CompiledMobilizePlan)
+      displayName: Display Compiled Mobilize File
```

Comment:
Did you ever get any use out of this step? If the previous step failed, then this step won't run, and if the previous step passed, then the compiled mobilize plan will be published in the report. I recommend removing this step.